### PR TITLE
initial definition and documentation for 'custom' placetype

### PR DESCRIPTION
**Pull request by straup**

### CODE REVIEW

**Review by stepps00 — APPROVED** [review]
Will need to run the `compile.py` script to update the spec docs, but LGTM otherwise.

File: README.md [hunk 1]
```diff
@@ -143,6 +143,16 @@ Basically places that issue passports, notwithstanding the details (like empires
 
 Continents are considered `common`.
 
+### custom
+
+`1729783759`
+
+Custom placetypes are provided as a convenience for projects producing data that live outside of the "core" Who's On First (WOF) data repositories as a way to allow those records to have a well-known WOF placetype. This ensures that these projects can continue to use the WOF-related tooling, some of which enforces strict validation on things like placetypes.
+
+The `custom` placetype has no predefined list of ancestors or any other computationally-derived metadata so properties like `wof:hierarchy` will need to be produced by the projects using the `custom` placetype on an as-needed basis. Any specific detail about a record's placetype should be encoded in a project-specific namespace.
```
Comment (nvkelso) [inline README.md:152]: Should we provide an example, like `(project:*)`

> Any project-specific properties for `custom` placetype records should be encoded in a project-specific namespace (eg: `project:*`).

**Review by nvkelso — APPROVED** [review]
One nit with wording, otherwise LGTM

File: README.md (hunk 1 above)
Comment (nvkelso) [reply README.md:152]: Potentially those would overlap with other wof-sources namespaces? Should tooling know to look for `custom` and then not assume the namespace will appear in wof-sources?

Comment (thisisaaronland) [reply README.md:152]: `custom` is not a namespace, only a placetype so I am not sure I understand the concern/question about overlapping namespaces. Can you elaborate.

I can add a link to the SFO Museum stuff for projects, as well as this which is the "long-long" version: https://mw19.mwconf.org/paper/mapping-space-time-and-the-collection-at-sfo-museum/

**Pull request merged**